datasync/downloader/peer.go

```diff
@@ -210,6 +220,28 @@ func (p *peerConnection) FetchReceipts(request *fetchRequest) error {
 	return nil
 }
 
+// FetchStakingInfo sends a staking information retrieval request to the remote peer.
+func (p *peerConnection) FetchStakingInfo(request *fetchRequest) error {
```

Comment:
Please, check whether this method is called when p.version < 65

Reply:
Please take a look https://github.com/klaytn/klaytn/pull/1394#discussion_r901229502